include/wasm.hh

```diff
@@ -689,11 +768,14 @@ public:
 // Table Instances
 
 class WASM_API_EXTERN Table : public Extern {
-public:
-  Table() = delete;
-  ~Table();
+  friend class Destroyer<Table>;
+  void destroy();
+
+protected:
+  Table() = default;
+  ~Table() = default;
 
-  using size_t = uint32_t;
```

Comment:
Ah, Table::size_t is not the same as C's size_t type. So removing this typedef is incorrect. If we think reusing the canonical type name is confusing, we should find a better name.

Reply:
Removing this was just a mistake on my part.